site/guide/state.md

```diff
@@ -85,6 +85,19 @@ fn from_request(req: &'a Request<'r>) -> request::Outcome<T, ()> {
 
 [`Request::guard()`]: https://api.rocket.rs/rocket/struct.Request.html#method.guard
 
+### Request-Local State
```

Comment:
What other parts of the guide should Request-Local State / Request-Local Cache be called out in?